src/SFA.DAS.RoATPService.Application.Api/Controllers/OrganisationController.cs

```diff
@@ -41,13 +41,13 @@ public async Task<IActionResult> Get(Guid organisationId)
         }
 
         [HttpPost]
-        [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(bool))]
+        [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(Guid?))]
         [SwaggerResponse((int)HttpStatusCode.BadRequest, typeof(IDictionary<string, string>))]
         [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
         [Route("create")]
         public async Task<IActionResult> Create([FromBody] CreateOrganisationRequest createOrganisationRequest)
         {
-            bool result = await _mediator.Send(createOrganisationRequest);
+            var result = await _mediator.Send(createOrganisationRequest);
 
             return Ok(result);
```

Comment:
Can we merge this into a single line? i.e.
return Ok(await _mediator.Send(.....